puffin/src/cli.rs

```diff
@@ -31,6 +31,7 @@ where
         .version(puffin::version())
         .author(crate_authors!())
         .about(title.as_ref().to_owned())
+        .arg(arg!(-T --target <T> "The PUT to use"))
```

Comment:
Using `--put` might be more clear?  Anyway at some point we will need to cleanup the CLI so I'm okay with `--target` for the moment. An opinion @LCBH?